src/tests/ftest/slurm_setup.py

```diff
@@ -55,6 +57,28 @@
     "grep -v \"^#\\w\" /etc/slurm/slurm.conf"]
 
 
+def create_epilog_script(args):
+    """Create epilog script to run after each job.
```

Comment:
pylint is complaining:
`wrong-spelling-in-docstring, Wrong spelling of a word 'epilog' in a docstring`
You could add the word to `utils/cq/words.dict`